Documentation/ceph-disaster-recovery.md

```diff
@@ -97,7 +97,7 @@ ceph-mon \
 Patch the `rook-ceph-mon-b` Deployment to stop this mon working without deleting the mon pod:
 
 ```console
-kubectl -n rook-ceph patch deployment rook-ceph-mon-b -p '{"op":"remove", "path":"/spec/template/spec/containers/0/livenessProbe"}'
+kubectl -n rook-ceph patch deployment rook-ceph-mon-b  --type='json' -p '[{"op":"remove", "path":"/spec/template/spec/containers/0/livenessProbe"}]'
 
 kubectl -n rook-ceph patch deployment rook-ceph-mon-b -p '{"spec": {"template": {"spec": {"containers": [{"name": "mon", "command": ["sleep", "infinity"], "args": []}]}}}}'
```

Comment:
Do we not need the same fix on this line?

Reply:
This one is fine.